src/utils/rootkeypackage_utils/inc/aduc/rootkeypackage_utils.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file rootkeypackage_utils.h
+ * @brief rootkeypackage_utils interface.
+ *
+ * @copyright Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+#ifndef ROOTKEYPACKAGE_UTILS_H
+#define ROOTKEYPACKAGE_UTILS_H
+
+#include "aduc/rootkeypackage_types.h"
```

Comment:
nit: sort header

Reply:
It's already clang-formatted.  If I touch/change it or swap the #include lines, clang-format sorts by putting `#include < ... >` after #include " ... "